fvm/src/utils.rs

```diff
@@ -0,0 +1,20 @@
+use kythera_common::abi::derive_method_num;
+
+pub const KYTHERA_NETWORK_ID: u64 = 1312;
+
+pub(crate) const WARP_METHOD: &str = "Warp";
+pub(crate) const EPOCH_METHOD: &str = "Epoch";
+pub(crate) const FEE_METHOD: &str = "Fee";
+pub(crate) const CHAIN_ID_METHOD: &str = "ChainId";
+pub(crate) const PRANK_METHOD: &str = "Prank";
+pub(crate) const TRICK_METHOD: &str = "Trick";
+
+lazy_static::lazy_static! {
```

Comment:
nit: `OnceCell` is [going to be stable soon](https://github.com/rust-lang/rfcs/pull/2788#issuecomment-1491194036) so it probably makes sense to [use its crate instead](https://crates.io/crates/once_cell)